src/components/Footer/SubComponents/PromotionBanner/PromotionBanner.tsx

```diff
@@ -0,0 +1,41 @@
+import { getContentfulPromotion } from '@/services/getContentfulPromotion'
+import Style from './PromotionBanner.module.scss'
+import Image from 'next/image'
+import parseTextWithSpan from '@/helpers/parseTextWithSpan'
+export default async function PromotionBanner() {
+  const promotion = await getContentfulPromotion()
+  if (!promotion) return
+  const { title, description, thumbnail, thumbnailTablet, thumbnailDesktop } =
+    promotion
+  return (
+    <div className='wrapper'>
+      <div className={Style['promotion']}>
+        <div className={Style['promotion__image']}>
+          <picture>
```

Comment:
any reason why we are using picture here? as far as I know this should be handled by the DAM in contentful?